Makefile

```diff
@@ -28,5 +32,13 @@ check: test
 	@test -x $(linter) || (echo "Please install linter from https://github.com/golangci/golangci-lint/releases/tag/v1.25.1 to $(HOME)/go/bin")
 	$(linter) run --build-tags vpn
 
+handlers:
```

Comment:
I think we could do this solely in yocto/meta-lmp.
I mean install `template-toml-update` to `${D}${datadir}/fioconfig/handlers` https://github.com/foundriesio/meta-lmp/blob/51caa20decd7cea2ceaeb169273ebcd089b45274/meta-lmp-base/recipes-support/fioconfig/fioconfig_git.bb#L34C70-L34C103 and then do the variable substitution after that.

Similar to what we do for the aklite service https://github.com/foundriesio/meta-lmp/blob/51caa20decd7cea2ceaeb169273ebcd089b45274/meta-lmp-base/recipes-sota/aktualizr/aktualizr_%25.bbappend#L40. The only difference is that we have to do it in some "do_postinstall" or "do_install_append".

Reply:
@mike-sul , yes, it's possible, I was trying to keep the retro compatibility with the code.

Reply:
I will revert the makefile changes and just keep the templates